test/jpypetest/test_collection.py

```diff
@@ -4,86 +4,84 @@
 import collections.abc
 
 
-class CollectionTestCase(common.JPypeTestCase):
+class CollectionCase(common.JPypeTestCase):
 
     def setUp(self):
-        super(CollectionTestCase, self).setUp()
+        common.JPypeTestCase.setUp(self)
```

Comment:
whats wrong with super?

Reply:
When I was copying and pasting around the test code and changing the method names the ``super`` ended up tripping me because it forced the name of the test case to match.  Thus I just switched to to match all of the the others.

If you would like me to use a script to make everything consistently use super instead I can also make that happen.